process/coordinator/process_test.go

```diff
@@ -2813,3 +3016,6 @@ func TestTransactionCoordinator_GetNumOfCrossShardSpecialTxsShouldWork(t *testin
 	}
 	assert.Equal(t, 1, getNumOfCrossShardScCallsOrSpecialTxs(mb, allTxs, 0))
 }
+
+//TODO: Unit tests for methods: VerifyCreatedMiniBlocks, verifyGasLimit, checkGasConsumedByMiniBlockInReceiverShard,
```

Comment:
Maybe add a Jira task so we won't forget about this?

Reply:
I will add them also in the current task anyway, in the next commit, but first I added only the productive code just to be easily to be reviewed